JeMPI_Apps/JeMPI_UI/src/services/ApiClient.ts

```diff
@@ -29,12 +27,46 @@ import {
   PatientRecord
 } from 'types/PatientRecord'
 import { Notifications } from 'types/Notification'
+import { Config } from 'config'
+import axios from 'axios'
+import { getCookie } from '../utils/misc'
 
-const client = config.shouldMockBackend ? moxios : axiosInstance
+export class ApiClient {
+  client!: AxiosInstance
+
+  async init(config: Config) {
+    if (process.env.NODE_ENV === 'test') {
```

Comment:
Should this not be `process.env.shouldMockBackend`, and opposed to overloading `NODE_ENV` (i.e what if want to set the enviroment to 'development' and still want to use the test server)

p.s as said earlier, I actually don't think this should be bundled in the production site. It creates unnecessary security risks. These process should run externally